src/ddmd/mars.d

```diff
@@ -245,6 +245,29 @@ extern (C++) void genCmain(Scope* sc)
     rootHasMain = sc._module;
 }
 
+private void semantic3OnDependencies(Module m, StringTable *importArray)
```

Comment:
This is incorrect.  What you are missing is that there are two other compiles (LDC and GDC) and that `mars.d` is not common to all three frontends.